nrf_dm: Adding version 0.7.0 of nrf_dm (#735)
nrfconnect/sdk-nrfxlib: nrf_dm/README.rst

Nordic Distance Measurement library
###################################

The Distance Measurement library measures the distance between nRF52 Series and nRF53 Series devices.
It does the following:

* It provides an easy-to-use interface for measuring distances.
* It computes the distance using all the information available to the device.
* It computes the distance based on the measured differential RF physical channel frequency response (MCPD mode) or the real time flight of packets (RTT mode).
* It performs all the mathematics necessary to compute the distance inside the library.

The library is available as hard-float builds for the nRF52 Series.

> On the nRF53 Series, the library is available as soft-float build for the application core.

Reviewer: network core

> On the nRF53 Series, the library is available as soft-float build for the application core.

Reviewer 2: Ah, then
```suggestion
On the nRF53 Series, the library is available as a soft-float build for the application core.
```

> For the network core of the nRF53 Series, there is a hard-float library which contains the computation heavy post-processing functionality only.

Reviewer 2: ```suggestion
For the network core of the nRF53 Series, there is a hard-float library that handles the computation-heavy post-processing functionality only.
```
I've also seen "compute-intensive" used. Would that be appropriate instead of "computation-heavy"?

Author: will change it 👍

> For the network core of the nRF53 Series, there is a hard-float library which contains the computation heavy post-processing functionality only.

Reviewer: application core
